src/api/libcellml/component.h

```diff
@@ -49,7 +48,109 @@ class LIBCELLML_EXPORT Component: public Nameable
      */
     void addComponent(const Component &c);
```

Comment:
The casing for the description of ```serialise()``` and ```addComponent()``` is not consistent with the other descriptions.